src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.ts

```diff
@@ -263,6 +268,38 @@ export class SnapshotListComponent {
     });
   }
 
+  doMultiDelete(selected){
+    let multiDeleteMsg = this.entityList.getMultiDeleteMessage(selected);
+    this.dialogService.confirm('Delete', multiDeleteMsg, false, T('Delete') ).subscribe((res) => {
+      if(res){
+        this.startMultiDeleteProgress(selected);
+      }
+    });
+  }
+
+  startMultiDeleteProgress(selected){
+    const params = this.wsMultiDeleteParams(selected);
+    let dialogRef = this.dialog.open(EntityJobComponent, { data: { title: T("Deleting Snapshots")}, disableClose: true });
+    dialogRef.componentInstance.setCall(this.wsMultiDelete, params);
+    dialogRef.componentInstance.progressNumberType = 'nopercent';
+    dialogRef.componentInstance.submit();
+
+    dialogRef.componentInstance.success.subscribe((job_res) => {
+      dialogRef.close();
+      this.entityList.getData();
+      this.entityList.selected = [];
+
+      let infoMessage: string = T('Deleted ') + params[1].length;
```

Comment:
Having leading/trailing spaces as part of the translation string is wrong, no?

Reply:
Updated